entry_points.ini

```diff
@@ -34,3 +34,4 @@
 #
 #   test = libcsm.bax:baz [bob, alice]
 [console_scripts]
+set_image_in_bss = libcsm.set_image_in_bss:main
```

Comment:
nit: since this will be the command the user invokes, it would be more conventional to switch to hyphens.